task/send-to-webhook-slack/0.1/send-to-webhook-slack.yaml

```diff
@@ -20,15 +21,32 @@ spec:
   - name: message
     type: string
     description: plain text message
+  - name: bot-name
+    type: string
+    description: plain text message
+    default: 'Tekton Bot'
+  - name: icon-emoji
+    type: string
```

Comment:
can we please update the README as well with these new params?

Reply:
Added docs in the README. Forget them :-(